cron/daily/100-pending-orders-reminder.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import models, { Op } from '../../server/models';
+import status from '../../server/constants/order_status';
+import * as libPayments from '../../server/lib/payments';
+
+const REMINDER_DAYS = 4;
+
+const fetchPendingOrders = async date => {
+  const orders = await models.Order.findAll({
+    where: {
+      status: status.PENDING,
+      deletedAt: null,
+      PaymentMethodId: null,
+      createdAt: { [Op.lte]: date },
+    },
+    include: [
+      { model: models.Collective, as: 'fromCollective' },
+      { model: models.User, as: 'createdByUser' },
+      { model: models.Collective, as: 'collective' },
+    ],
+  });
+
+  return orders;
+};
+
+const run = async () => {
+  //
+  const reminderDate = process.env.START_DATE ? new Date(process.env.START_DATE) : new Date();
+  reminderDate.setDate(reminderDate.getDate() - REMINDER_DAYS);
+
+  const orders = await fetchPendingOrders(reminderDate);
```

Comment:
My understanding is that this will send a reminder email for each pending order every day during 4 days, while what we want is to send **only one** email after 4 days. Am I missing something?

Reply:
No, It's a misunderstanding from me. I fixed it.